README.md

```diff
@@ -181,6 +184,82 @@ Events caused by pointer movement or scrolling do not count as "input" for the
 purpose of the recent input exclusion and the input-related attributes on the
 `LayoutShift` entry.
 
+### Source Attribution
+
+_NOTE: The `sources` attribute is not yet implemented in any browser, and its
+definition is still under active development._
+
+On a complex website, it can be difficult to understand the cause of a high
+CLS score given only the numeric values in the `value` attribute of the
+`LayoutShift` entries.
+
```

Comment:
Do we want these sentences to all be a separate paragraph?

Reply:
I think each paragraph in the Source Attribution section has a distinct focus.  The first is a prefatory note about implementation status; the second gives our motivation; the third presents the `sources` attribute and its overarching purpose; the fourth introduces the `LayoutShiftAttribution` interface.  Lumping it all into one paragraph would muddy that structure, and make the message harder to digest.

But I will make a small tweak to the third paragraph so that it flows more smoothly from the second.